### PR TITLE
adding separator between groups in user list

### PR DESCRIPTION
**Pull request by alagroy-42**
solves #354

### CODE REVIEW

File: lib/jelix-admin-modules/jacl2db_admin/controllers/users.classic.php [hunk 1]
```diff
@@ -61,6 +59,7 @@ public function index()
         $offset = $this->param('idx', 0, true);
         $grpid = $this->param('grpid', jAcl2DbAdminUIManager::FILTER_GROUP_ALL_USERS, true);
         $filter = trim($this->param('filter'));
+        $last = count($groups) - 3;
```
Comment (laurentj) [inline lib/jelix-admin-modules/jacl2db_admin/controllers/users.classic.php:62]: why `-3`? I guess it should be `-1` right?

**Review by laurentj — CHANGES_REQUESTED** [review]

File: lib/jelix-admin-modules/jacl2db_admin/controllers/users.classic.php (hunk 1 above)
Comment (alagroy-42) [reply lib/jelix-admin-modules/jacl2db_admin/controllers/users.classic.php:62]: `$groups` is pre-populated with 2 groups that will never be displayed, lines 43 and 49.

**Review by laurentj — APPROVED** [review]

File: lib/jelix-admin-modules/jacl2db_admin/controllers/users.classic.php (hunk 1 above)
Comment (laurentj) [reply lib/jelix-admin-modules/jacl2db_admin/controllers/users.classic.php:62]: ok I see

**Comment by laurentj** [comment]
thank you for your patch

**Pull request merged**